bower.json

```diff
@@ -17,7 +17,7 @@
   ],
   "repository": {
     "type": "git",
-    "url": "git://github.com/paf31/purescript-canvas.git"
+    "url": "https://github.com/paf31/purescript-canvas.git"
```

Comment:
Should this be updated to `purescript-web1` rather than `paf31`?

Reply:
ping @thomashoneyman

Reply:
Oops, sorry. Looks like this package is here in the registry:

https://github.com/purescript/registry/blob/d00db678819f73fbeefbe34798ff48a69fd443b8/bower-packages.json#L269

...which seems to be `purescript/` rather than `purescript-web/`. @f-f may know for sure, but I think this URL just needs to match the registry so we can publish new versions.